app/components/elements/forms/field_component.rb

```diff
@@ -4,9 +4,9 @@ module Elements
   module Forms
     # Base component for all form fields.
     class FieldComponent < ApplicationComponent
-      def initialize(form:, field_name:, required: false, hidden_label: false, label: nil, help_text: nil, # rubocop:disable Metrics/ParameterLists
-                     disabled: false, hidden: false, data: {}, placeholder: nil, width: nil, label_classes: [],
-                     container_classes: [], input_classes: [])
+      def initialize(form:, field_name:, required: false, hidden_label: false, label: nil, help_text: nil, # rubocop:disable Metrics/ParameterLists, Metric/MethodLength
+                     disabled: false, hidden: false, data: {}, input_data: {}, placeholder: nil, width: nil,
```

Comment:
There was an inconsistency about whether `data` was for the container or the input. This makes it consistent.